docs/docs/visual-testing-with-storybook.md

```diff
@@ -7,30 +7,31 @@ Knowing your components look as intended in every permutation is not only a grea
 1. know what components are available to them in a given project and
 2. what props those components accept and what all of the states of that component are.
 
-As your project grows over time having this information available will be invaluable. This is the function of the Storybook library. Storybook is a UI development environment for your UI components. With it, you can visualize different states of your UI components and develop them interactively.
+As your project grows over time having this information available will be invaluable. This is the function of the [Storybook](https://storybook.js.org/) library. Storybook is a UI development environment for your UI components. With it, you can visualize different states of your UI components and develop them interactively.
 
 ## Setting up your environment
 
-To set up Storybook you need to install dependencies and do some custom configuration. First, install the Storybook CLI.
+To set up Storybook run the following command inside you project folder:
 
 ```sh
-npm install -g @storybook/cli
+npx -p @storybook/cli sb init
 ```
 
-Once the CLI is installed, the next step is to run the `sb init` command that is now available from the root directory of your Gatsby project.
-
-```sh
-cd my-awesome-gatsby-project
-sb init
-```
-
-> Note that if you're running a recent version of `npm` (5.2.0+) you can run the following single command instead: `npx -p @storybook/cli sb init`, which is the recommended method by Storybook. This doesn't install the CLI on your machine, thereby ensuring you're always running the latest version of the CLI.
+> Note that this is the recommended method by Storybook. If you are running an older version of npm (< 5.2.0) you must install the Storybook CLI globally:
+>
+> ```sh
+> npm install -g @storybook/cli
+> cd my-awesome-gatsby-project
+> sb init
+> ```
 
 The `sb init` command bootstraps the basic config necessary to run Storybook for a React project. However, since this is for a Gatsby project, you need to update the default Storybook configuration a bit so that you don't get errors when trying to use Gatsby specific components inside of the stories.
 
 To update your Storybook config open `.storybook/config.js` and add the following before the `configure` method at the bottom of the file.
 
-```js
+```js{3-17}:title=.storybook/config.js
```

Comment:
This is basically the entire file right? I don't think this is valuable if that's the case.

Reply:
I made some changes to this snippet, and now the highlight seems more important